spotify/spotify.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Add ~/.config to the XDG_CONFIG_HOME env var
+XDG_CONFIG_HOME=${XDG_CONFIG_HOME:-~/.config}
+
+# Allow users to override command-line options
+if [[ -f "${XDG_CONFIG_HOME}/spotify-flags.conf" ]]; then
+    SPOTIFY_USER_FLAGS="$(grep -v '^#' "${XDG_CONFIG_HOME}/chrome-flags.conf")"
```

Comment:
We could maybe print the flags to so it is included in the startup log should custom flags be defined

Reply:
Good idea, done in https://github.com/christian-heusel/aur/pull/4/commits/e1e10cd446abe06fcd9184d45d3a9eb51fcff7e5